Docs/user_guide/modules/transforms.md

```diff
@@ -19,8 +19,9 @@ Supported transform types:
 
 Transform node can be created in multiple ways:
 - Method A: In Data module's Subject hierarchy tab, right-click on the "Transform" column and choose "Create new transform". This always creates a general "Transform".
-- Method B: In Data module's Transform hierarchy tab, right-click on an item and choose "Insert transform". This always creates a "Linear transform". Advantage of this method is that it is easy to build and overview hierarchy of transforms.
-- Method C: In Transforms module click on "Active transform" node selector and choose one of the "Create new..." options.
+- Method B: In Data module's Subject hierarchy tab, right-click on the "Visibility" column of any transformable node and choose "Interaction" to create a parent transform and display the interaction handles. This always creates a "Linear transform"
```

Comment:
For all new developments we should use general Transform instead of linear (since the transform type can change as you harden transforms on each other, using class name to distinguish between transforms is not a good solution). See https://github.com/Slicer/Slicer/issues/5718

Reply:
The node created is a linear "vtkMRMLTransformNode", so no changes should be needed in the code.